src/main/java/spoon/reflect/visitor/CommentHelper.java

```diff
@@ -34,7 +34,7 @@
 /**
  * Computes source code representation of the Comment literal
  */
-class CommentHelper {
+public class CommentHelper {
```

Comment:
it is needed in tests, which are in different package.
May be I can remove part of the new tests, because the DJPP is already well tested by many other tests. Then CommentHelper can stay package protected

Reply:
> May be I can remove part of the new tests, because the DJPP is already well tested by many other tests. Then CommentHelper can stay package protected

OK. This would be consistent with the rest.